### PR TITLE
Add Classic token filter docs

### PR DESCRIPTION
**Pull request by AntonEliatra**
### Description
Add Classic token filter docs

### Issues Resolved
Closes https://github.com/opensearch-project/documentation-website/issues/7876

### Version
all

### Checklist
- [x] By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license and subject to the [Developers Certificate of Origin](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

**Comment by kolchfa-aws** [comment]
@udabhas Could you review this PR for technical accuracy?

**Comment by vagimeli** [comment]
@varun-lodaya The documentation is awaiting tech review and approval, which is delaying progress. Could you please suggest alternative reviewers who can assist with this task in a timely manner? We're eager to move this forward. Thank you.

File: _analyzers/token-filters/classic.md [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+title: classic
+parent: Token filters
+nav_order: 50
+---
+
+# Classic token filter
+
+The primary function of the classic token filter is to work alongside the classic tokenizer. It processes tokens by applying several common transformations. These transformations aid in text analysis and search. These include:
+ - Removal of possessive endings such as *'s*, for example: *John's* becomes *John*.
+ - Separating words on internal hyphens, making terms like *co-operate* become tokens *co* and *operate*.
```
Comment (udabhas) [inline _analyzers/token-filters/classic.md:12]: I donot think this statement is correct wrt to classic token filter. 

I believe words are already separated when input is streamed through classic tokenizer not when passed through classic token filter. 

Below example shows words separation done via tokenizer

```
PUT my-index-classic2
{
  "settings": {
    "analysis": {
      "analyzer": {
        "custom_classic": {
          "type": "custom",
          "tokenizer": "classic"
        }
      }
    }
  }
}

GET my-index-classic2/_analyze
{
  "analyzer": "custom_classic", 
  "text": "John's co-operate was excellent."
}
```
Response
```
{
  "tokens": [
    {
      "token": "John's",
      "start_offset": 0,
      "end_offset": 6,
      "type": "<APOSTROPHE>",
      "position": 0
    },
    {
      "token": "co",
      "start_offset": 7,
      "end_offset": 9,
      "type": "<ALPHANUM>",
      "position": 1
    },
    {
      "token": "operate",
      "start_offset": 10,
      "end_offset": 17,
      "type": "<ALPHANUM>",
      "position": 2
    },
    {
      "token": "was",
      "start_offset": 18,
      "end_offset": 21,
      "type": "<ALPHANUM>",
      "position": 3
    },
    {
      "token": "excellent",
      "start_offset": 22,
      "end_offset": 31,
      "type": "<ALPHANUM>",
      "position": 4
    }
  ]
}
```

**Comment by vagimeli** [comment]
@AntonEliatra Please see the tech reviewer's feedback. Thank you.

File: _analyzers/token-filters/classic.md (hunk 1 above)
Comment (AntonEliatra) [reply _analyzers/token-filters/classic.md]: This is correct, I have removed this now

**Comment by vagimeli** [comment]
@AntonEliatra Thanks for address tech comments. Moving this forward for doc review with @kolchfa-aws.

**Review by udabhas — APPROVED** [review]
Doc Looks good to me

File: _analyzers/token-filters/classic.md [hunk 2]
```diff
@@ -0,0 +1,93 @@
+---
+layout: default
+title: classic
+parent: Token filters
+nav_order: 50
+---
+
+# Classic token filter
+
+The primary function of the classic token filter is to work alongside the classic tokenizer. It processes tokens by applying several common transformations. These transformations aid in text analysis and search. These include:
```
Comment (kolchfa-aws) [inline _analyzers/token-filters/classic.md]:
```suggestion
The primary function of the classic token filter is to work alongside the classic tokenizer. It processes tokens by applying the following common transformations, which aid in text analysis and search:
```

**Review by kolchfa-aws — APPROVED** [review]
LGTM. Thank you, @AntonEliatra!

File: _analyzers/token-filters/classic.md [hunk 3]
```diff
@@ -0,0 +1,93 @@
+---
+layout: default
+title: classic
```
Comment (natebower) [inline _analyzers/token-filters/classic.md:3]: Is this intentionally lowercase?

File: _analyzers/token-filters/classic.md [hunk 4]
```diff
@@ -0,0 +1,93 @@
...
+parent: Token filters
+nav_order: 50
+---
+
+# Classic token filter
+
+The primary function of the classic token filter is to work alongside the classic tokenizer. It processes tokens by applying the following common transformations, which aid in text analysis and search:
+ - Removal of possessive endings such as *'s*, for example: *John's* becomes *John*.
```
Comment (natebower) [inline _analyzers/token-filters/classic.md:11]:
```suggestion
 - Removal of possessive endings such as *'s*. For example, *John's* becomes *John*.
```

File: _analyzers/token-filters/classic.md [hunk 5]
```diff
@@ -0,0 +1,93 @@
...
+ - Removal of periods from acronyms, for example: *D.A.R.P.A.* becomes *DARPA*.
```
Comment (natebower) [inline _analyzers/token-filters/classic.md:12]:
```suggestion
 - Removal of periods from acronyms. For example, *D.A.R.P.A.* becomes *DARPA*.
```

**Review by natebower — COMMENTED** [review]
@AntonEliatra @kolchfa-aws Just a few changes. Thanks!

File: _analyzers/token-filters/classic.md (hunk 3 above)
Comment (kolchfa-aws) [reply _analyzers/token-filters/classic.md:3]: Should be uppercase, thanks for catching!

Comment (kolchfa-aws) [inline _analyzers/token-filters/classic.md:3]:
```suggestion
title: Classic
```

**Pull request merged**